jumanji/environments/combinatorial/routing/env_viewer.py

```diff
@@ -64,6 +64,7 @@ def __init__(
 
         for _ in range(num_agents):
             color = rnd.randint(0, 192, 3)
+            color = list(map(int, color))
             self.palette.append((color[0], color[1], color[2]))
```

Comment:
```suggestion
            self.palette.extend(map(int, color))
```

Reply:
Would have been simpler, but I get a mypy error.
```
jumanji/environments/combinatorial/routing/env_viewer.py:69: error: Argument 1 to "append" of "list" has incompatible type "Tuple[int, ...]"; expected "Tuple[int, int, int]"
```
Unless you see a way around it, I will stick to the slightly more verbose solution.

Reply:
I found a way around it based on your suggestion.
```
            r, g, b = map(int, rnd.randint(0, 192, 3))
            self.palette.append((r, g, b))
```
Agree to go with this?